app/models/user.rb

```diff
@@ -1,3 +1,14 @@
+# == Schema Information
```

Comment:
I recomend to configure the annotate gem to annotate at the end of the file. Keep the file header cleaner

Reply:
Fixed